clients/lsp-elixir.el

```diff
@@ -105,6 +105,21 @@ Leave as default to let `executable-find' search for it."
   :type '(repeat string)
   :package-version '(lsp-mode . "7.1"))
 
+
+(defconst lsp-elixir-ls-server-dir
+  (f-join lsp-server-install-dir "elixir-ls")
+  "Elixir-ls local server Directory")
+
+(defcustom lsp-elixir-local-server-command
+  (let ((file (f-join lsp-elixir-ls-server-dir
```

Comment:
This checks for file existence whenever lsp's clients are loaded, which make `lsp-mode` loads slower when user is not using `lsp-elixir`.
Can you move this default value to somewhere around here?
``` elisp
                                     `(,(or lsp-elixir-local-server-command
                                            (or (executable-find
                                                 (cl-first lsp-elixir-server-command))
                                                (lsp-package-path 'elixir-ls)))
```

Reply:
oh, my bad I forgot about that.